src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,94 @@
+package duke.task;
+
+import java.io.Serializable;
+
+/**
+ * This Task class represents a task.
+ */
+public class Task implements Serializable, Comparable<Task> {
+    private static final long serialVersionUID = 100;
```

Comment:
Since this is a constant, may want to name it all uppercase using underscore to separate words e.g. `SERIAL_UID`

Reply:
`serialVersionUID` is a special variable found in the Serializable Interface that helps to determine the Class version. Thus, the name cannot change without interfering with the function of the Class.